services/movie.py

```diff
@@ -0,0 +1,48 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids)
+
+    elif genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    elif actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+

```

Comment:
This check is redundant
 
` if genres_ids and actors_ids:
` 
 Instead of using elif, you can use separate if statements for each condition. 
 
 This approach will apply both filters to the queryset if both genres_ids and actors_ids are present. If only one is present, only the corresponding filter is applied. This way, you don't need the elif statements, and the code is simplified while still achieving the same result.